scipy/diff/__init__.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import division, print_function, absolute_import
+
+from _derivative import *
```

Comment:
these imports are the issue. It should be ``from ._derivative import *`` (note the ``.``)

and the same on the next two lines